packages/datadog-plugin-amqp10/test/integration-test/client.spec.js

```diff
@@ -13,36 +13,38 @@ describe('esm', () => {
   let proc
   let sandbox
 
-  before(async function () {
-    this.timeout(20000)
-    sandbox = await createSandbox(['amqp10', 'rhea'], false, [
-      `./packages/datadog-plugin-amqp10/test/integration-test/*`])
-  })
-
-  after(async () => {
-    await sandbox.remove()
-  })
+  withVersions('amqp10', 'amqp10', version => {
+    context('amqp10', () => {
```

Comment:
Don't need this additional context wrap. The `withVersions` call produces its own already so this just adds an extra level of depth for no reason.